core-bundle/src/Command/MigrateCommand.php

```diff
@@ -54,6 +55,7 @@ protected function configure(): void
             ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Show pending migrations and schema updates without executing them.')
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt, ndjson)', 'txt')
             ->addOption('no-backup', null, InputOption::VALUE_NONE, 'Disable the database backup which is created by default before executing the migrations.')
+            ->addOption('no-check', null, InputOption::VALUE_NONE, 'Disable checking for configuration errors.')
```

Comment:
Why this? The check could not be disabled in the install tool, either.

Reply:
I figured it would be a good idea to be able to skip checks if not needed. Same with the `--no-backup` option.

It also allows to disable checks when testing the migrate functionality.

Reply:
Not sure about this. Not wanting a backup is a whole different story than suppressing warnings about issues that need to be fixed and that previously blocked the installation process!

Reply:
Removed the option in 3d9d5e7ae875efce059c540050d3b61686cd1373.